src/schema/v2/artist/helpers.ts

```diff
@@ -190,3 +195,21 @@ export const getAuctionRecord = async (artist, auctionLotsLoader) => {
 
   return highAuctionRecord
 }
+
+const getRecentShow = (artist): string[] => {
```

Comment:
**suggestion [blocking]:** export function and test directly

Same feedback from me on this - I think these helper functions can be exported and tested directly with some happy path tests on the outer loop.

*format via [Conventional Comments](https://conventionalcomments.org/)*